src/clients/c++/experimental_api_v2/library/common_utils.h

```diff
@@ -72,4 +72,52 @@ class DECLSPEC Error {
   std::string msg_;
 };
 
+//==============================================================================
+/// Structure to hold options for Inference Request.
```

Comment:
Seems like these can be combined into common.h

Reply:
Fixed